int16 left_speed
float32 left_duration
int16 right_speed
float32 right_duration
int16 Hz